client/src/components/Behodler/Swap/TradingBox3/index.tsx

```diff
@@ -534,7 +534,9 @@ export default function (props: {}) {
     const balanceCheck = async (menu: string) => {
         const balanceResults = await FetchBalances(account || "0x0", contracts)
         let balances: TokenBalanceMapping[] = tokenDropDownList.map(t => {
-            let hexBalance = balanceResults.results[t.name].callsReturnContext[0].returnValues[0].hex.toString()
+            let hexBalance = balanceResults
+                ? balanceResults.results[t.name].callsReturnContext[0].returnValues[0].hex.toString()
+                : "0x00"
```

Comment:
this prevents the app from crashing completely when the token balances fail to load